src/SWP/WebRendererBundle/Routing/Loader/PagesLoader.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+/**
+ * This file is part of the Superdesk Web Publisher Web Renderer Bundle
+ *
+ * Copyright 2015 Sourcefabric z.u. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2015 Sourcefabric z.ú.
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\WebRendererBundle\Routing\Loader;
+
+use Doctrine\ORM\EntityManager;
+use Symfony\Component\Routing\RouteCollection;
+use Symfony\Component\Routing\Route;
+use Symfony\Component\Config\Loader\Loader;
+
+/**
+ * Pages Loader loads routes from a Pages entries.
+ *
+ * You can needt to configure loader in routing.yml file
+ *
+ * pages:
+ *     type: pages
+ */
+class PagesLoader extends Loader
+{
+    /**
+     * @var \Doctrine\ORM\EntityManager
+     */
+    protected $em;
```

Comment:
well, its not only to satisfy mass detector (althought it was created for some reason), its rather the readability and a way on how to understand what this variable is meant for by just looking on its name. We know what does it mean, someone else may not know, 2 chars variables means nothing. Doctrine documentation uses `$entityManager` variable name in examples. I know I am bitching but details matter (anyway its just a suggestion) :)